pkg/minikube/download/preload.go

```diff
@@ -116,36 +139,23 @@ func PreloadExists(k8sVersion, containerRuntime, driverName string, forcePreload
 	}
 
 	// If the preload existence is cached, just return that value.
-	if preloadState != preloadUnknown {
-		return preloadState == preloadPresent
+	cRuntimes, ok := preloadStates[k8sVersion]
```

Comment:
Could do `preloadState, ok := preloadStates[k8sVersion][containerRuntime]` to make it simpler

Reply:
Done.